hibp/table_hibp_password.go

```diff
@@ -4,29 +4,32 @@ import (
 	"context"
 	"crypto/sha1"
 	"fmt"
+	"strconv"
+	"strings"
 
 	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
 	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
 	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"github.com/wneessen/go-hibp"
 )
 
 type psswrdRow struct {
-	Hash  string
-	Count int64
+	hibp.Match
+	HashPrefix string
 }
 
 func tableHIBPPassword() *plugin.Table {
 	return &plugin.Table{
 		Name:        "hibp_password",
 		Description: "Password (hashes) tracked by HIBP.",
 		List: &plugin.ListConfig{
-			KeyColumns: plugin.AnyColumn([]string{"plaintext", "hash"}),
+			KeyColumns: plugin.AnyColumn([]string{"plaintext", "hash", "hash_prefix"}),
 			Hydrate:    listPasswords,
 		},
 		Columns: []*plugin.Column{
 			{Name: "plaintext", Type: proto.ColumnType_STRING, Transform: transform.FromQual("plaintext").NullIfZero(), Description: "The plain-text of the compromised password (sent as a hash to the API)."},
 			{Name: "hash", Type: proto.ColumnType_STRING, Description: "The hash of the compromised password."},
-			{Name: "count", Type: proto.ColumnType_INT, Description: "The total number of times this password has been found compromised."},
```

Comment:
Why did we remove this column? From what I recall, this column returned results ok?